local-dev/templates/docker-rsync-local-rawls.sh

```diff
@@ -150,6 +150,11 @@ start_server () {
     docker start sqlproxy
     echo "Starting proxy..."
     docker start rawls-proxy
+     # Check if rawls-proxy started successfully
```

Comment:
rawls-proxy was failing quietly and I didn't notice - I've added this in so it wouldn't continue to run if the proxy failed, but haven't been able to reproduce a failure to see if it even works.  Can remove, or welcome comments on a better check

Reply:
I'm not opposed, but all of the containers (rawls-proxy, rawls-sbt, sqlproxy, rawls-rsync-container) are necessary. I think we should check all of them or none of them to be consistent.

Reply:
Ok I added one for sqlproxy - I think for the last one you'll see in the output any failures